srcpkgs/teckit/template

```diff
@@ -0,0 +1,44 @@
+# Template file for 'teckit'
+pkgname=teckit
+version=2.5.8
+revision=1
+build_style=gnu-configure
+hostmakedepends="libtool autoconf automake"
+makedepends="zlib-devel"
+depends="libteckit"
+short_desc="A Text Encoding Conversion toolkit"
+maintainer="svenper <svenper@tuta.io>"
+license="custom"
+homepage="http://scripts.sil.org/TECkitIntro"
+changelog="http://scripts.sil.org/TECkitRecentChanges"
+distfiles="https://github.com/silnrsi/teckit/archive/v${version}.tar.gz"
+checksum=0ea52b1304f430aaebff99fec355bc7e4ad75b16ba58959fae5079627f925f93
+
+pre_configure() {
+	./autogen.sh
+}
+
+post_install() {
+	vlicense license/LICENSING.txt
+}
+
+libteckit_package() {
+	short_desc+=" - library"
+	pkg_install() {
+		vmove "usr/lib/libTECkit.so.*"
+		vmove "usr/lib/libTECkit_Compiler.so.*"
+	}
+}
+
+libteckit-devel_package() {
+	depends="libteckit>=${version}_${revision}"
+	short_desc+=" - library development files"
+	pkg_install() {
+		vmove "usr/include/teckit"
+		vmove "usr/lib/pkgconfig/teckit.pc"
+		vmove "usr/lib/libTECkit.la"
```

Comment:
we do not want libtool files without good reason

Reply:
oops, `.a`, not `.la`